### DISCUSSION TITLE
【求助】新艾利都的出境申请好批吗

### DISCUSSION THREAD

**Discussion by March-777th** [share121/inter-knot#3213] [其他]
<img width="1066" alt="Snipaste_2024-08-20_00-46-18" src="https://github.com/user-attachments/assets/d639496a-f187-47bd-afd5-32accbb116e0">
最近兄弟找我有急事，地点在新艾利都外，需要准备什么手续？<img src="https://bbs-static.miyoushe.com/static/2024/03/21/dab9bdd5012200a2902c8ae19dc1c8c9_4053703670259707891.png" alt="丽都漫步-可琳" width="75" >

**Comment by Maixiaofanqie** [comment]
不知道，但是好像很多人的出境明天就能批下来<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
